content/blog/2022-07-28-switching-to-dvc-from-jupyter-vscode.md

```diff
@@ -0,0 +1,154 @@
+---
+title: Converting a Jupyter Notebook to a DVC Project
+date: 2022-07-28
+description: >
+  Working with notebooks is common in machine learning. That's why we're
+  covering some tools that make it easy to do more with a complex project.
+descriptionLong: >
+  Once you've run some experiments in a Jupyter notebook, you know that you
+  can't save each experiment. Now, if you're using the Jupyter VSCode extension,
+  we can show you how to make those experiments reproducible with the addition
```

Comment:
I'm not sure the blog actually covers this. It shows the comparison table, but not reproducing experiments.

Reply:
I added a bit to show where to run the experiments in VS Code, but let me know if you think it needs more.